src/api/endpoints/analytics.py

```diff
@@ -0,0 +1,11 @@
+from fastapi import APIRouter, Depends
+
+from src.api.schemas import Analytic
+from src.api.services.analitics import AnalyticsService
+
+analytic_router = APIRouter()
+
+
+@analytic_router.get("/", response_model=Analytic, description="Возращает статистику сервиса.")
```

Comment:
Рекомендую поставить spell checker, если работаешь в vscode, будет подсвечивать все подобные ошибки)

```suggestion
@analytic_router.get("/", response_model=Analytic, description="Возвращает статистику сервиса.")
```